account/user_blackbox_test.go

```diff
@@ -95,7 +95,8 @@ func (s *userBlackBoxTest) TestExistsUser() {
 		// Check not existing
 		err := s.repo.CheckExists(s.ctx, uuid.NewV4().String())
 		// then
-		require.IsType(s.T(), &goa.ErrorResponse{}, err)
+		var errorType errors.NotFoundError
+		require.IsType(s.T(), errorType, err)
```

Comment:
the problem was that my editor was nicely-secretly adding almighty pkgs for the errors in the import instead of f8-wit errors pkg.